code/pilotfile/plr.cpp

```diff
@@ -772,7 +773,7 @@ void pilotfile::plr_write_controls()
 			handler->writeInt("axis_map", -1);
 		}
 		
-		handler->writeInt("invert_axis", item.is_inverted() ? 1 : 0);
+		handler->writeInt("invert_axis", (item.first.is_inverted() || item.second.is_inverted()) ? 1 : 0);
```

Comment:
Yes, I changed the logic from `&&` to `||` here, as I think this better handles the case of saving the old data when the player has two different inversions.